.github/workflows/reusable_sycl.yml

```diff
@@ -120,3 +122,24 @@ jobs:
         echo "---Run build_log test" && ./build_log
         echo "---Run parallel_for_range_roundup test" && ./parallel_for_range_roundup
         echo "---Run fill_any_size test" && ./fill_any_size
+
+    # Run sycl benchmarks
```

Comment:
run this on benchmark machine

Reply:
Done, extended the Nightly benchmark job instead